lib/menu.rb

```diff
@@ -0,0 +1,11 @@
+class Menu
+
+  attr_reader :food_options
+
+  def food_options
+    dishes = { 'Spring rolls' => 2.50,
```

Comment:
Could the dishes be extracted to a constant so that it can be changed without changing code?